webpack.config.js

```diff
@@ -3,7 +3,7 @@
 // Basics
 const path = require('path'); // Cross platform path resolver
 const webpack = require('webpack');
-require('dotenv').config({ path: './.env' });
+const envVariables = require('dotenv').config({ path: './.env' }).parsed;
```

Comment:
todo: You also need to update `.gitignore` in the project root. Else, the `.env` file will be commited, the next time someone adds it.

```
# nvm
.nvmrc

+ # Project
+ .env
```